_documentation/developer/bzfs_api_functions/bz_APIVersion.md

```diff
@@ -5,7 +5,7 @@ signatures:
     - []
 returns:
     dataType: int
-    description: ~
+    description: BZAPI version.
 ---
 
-This function needs a description. Please feel free to contribute a description to this function.
+Useful for checking API version requirements a plug-in may have.
```

Comment:
I'd prefer if the description was objective about what this function did rather than when it'd be useful. Also, I'd mention that the API version is mainly used so bzfs can safely fail to load plugins compiled with an older version of bzfs.

Someone please correct me on this fact because `BZ_API_VERSION` shouldn't change unless something drastically changes in the API.

Reply:
@allejo 
I'm not sure, but I believe that 'any' time the API changes, the BZ_API_VERSION changes.
This is going from memory though, so I may be mistaken on this.

@The-Noah 
'**Useful for checking**' could perhaps be changed to '**Returns**'?
This would also provide additional consistency with the rest of your documentation as well.

Reply:
No, no, no. Like I said, `BZ_API_VERSION` does not update often (last change was 7 years ago, https://github.com/BZFlag-Dev/bzflag/commit/821bad9d13fec3a73c50da8c40c09ceab4423f6b). It's only changed when there's a protocol break, existing function signatures change, or API objects' internals change. These changes would cause problems when bzfs tries to load a plugin that was previously compiled with a different `BZ_API_VERSION`.

Reply:
Did you see the new commit?